pkg/controller/constrainttemplate/constrainttemplate_controller.go

```diff
@@ -106,6 +107,9 @@ func (a *Adder) InjectGetPod(getPod func() (*corev1.Pod, error)) {
 	a.GetPod = getPod
 }
 
+func (a *Adder) InjectMutationCache(mutationCache *mutation.System) {
```

Comment:
nit: one line

Reply:
Done

Reply:
two lines

Reply:
Forgot to push changes, should be better now